packages/sdks/javascript/src/DevopnessApiClient.ts

```diff
@@ -118,4 +121,17 @@ export class DevopnessApiClient {
 
     ApiBaseService.accessToken = accessToken;
   }
+
+  /**
+   *
+   * @summary Sets the callback function to be executed when the access token expires.
+   * @static
+   * @param {function(string)} callback - The callback function to be executed.
+   * This function should accept a single string parameter representing the expired access token.
+   */
+  public set onTokenExpired(callback: (accessToken: string) => void) {
+    if (callback) {
+      ApiBaseService.onTokenExpired = callback;
+    }
```

Comment:
@dudunog why use a conditional block here? `callback` is non-optional by the types on the function declaration above.

```suggestion
    ApiBaseService.onTokenExpired = callback;
```

Reply:
Ops, this makes sense, we don't need a conditional block there.